unmock/core/http.py

```diff
@@ -0,0 +1,143 @@
+from typing import Optional, List
+import http.client
+from urllib.parse import urlsplit, SplitResult
+
+from .options import UnmockOptions
+from .utils import Patchers
+
+__all__ = ["initialize", "reset"]
+
+# Backup:
+UNMOCK_AUTH = "___u__n_m_o_c_k_a_u_t__h_"
+PATCHERS = Patchers()
+STORIES = list()
+
+def parse_url(url) -> SplitResult:
+    parsed_url = urlsplit(url)
+    if parsed_url.scheme == "" or parsed_url.netloc == "":
+        return urlsplit("http://{url}".format(url=url))
+    return parsed_url
+
+def initialize(unmock_options: UnmockOptions, story: Optional[List[str]] = None, token: Optional[str] = None):
+    """Entry point to mock the standard http client. Both `urllib` and `requests` library use the
+    `http.client.HTTPConnection`, so mocking it should support their use aswell.
```

Comment:
Is this documentation meant for library users or developers? I'd expect the docstring for `initialize` to be more like "Initializes unmock library for capturing API calls. Params: ..."

Reply:
Good point! Meant for developers, will update.